python/cudf/cudf/core/dataframe.py

```diff
@@ -1780,6 +1780,14 @@ def _mimic_inplace(self, result, inplace=False):
         else:
             return result
 
+    def rank(self, axis=0, method="average", numeric_only=None, na_option="keep", ascending=True, pct=False):
+        """
+        Compute numerical data ranks (1 through n) along axis.
+
+        By default, equal values are assigned a rank that is the average of the ranks of those values.
+        """
+        return super().rank(axis, method, numeric_only, na_option, ascending, pct)
```

Comment:
Since we're not specializing `rank` in any way for `DataFrame`, we don't need this call to `super()`. We can get rid of DataFrame level `rank` entirely - it will just get inherited from `Frame`.